nat-lab/deps/Dockerfile

```diff
@@ -80,10 +82,17 @@ RUN \
     sed -i 's/\/sbin\/iptables/\/usr\/sbin\/iptables/g' /etc/upnpd.conf
 
 RUN \
-    ./install_stun_server.sh
+    ./install_stun_server.sh && \
+    rm install_stun_server.sh
```

Comment:
Why this `rm` is needed here?

Reply:
Because the script is used only once on build and never again.